english/marshmallow.md

```diff
@@ -19,6 +19,8 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 [Click](vote-the-president/hillary/Hillary.md) here to vote for Hillary Clinton.
 
+These options depress you, so you decide not to vote at all, and instead to [write a symphony](write-a-symphony/symphony.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.